### ISSUE TITLE
提供模型性能异常

### ISSUE THREAD

**Issue by TanLo** [dimitymiller/cac-openset#9]
作者你好，你这边提供的TinyImageNet模型，闭集和开集上均无法达到论文性能

**Comment by dimitymiller** [comment]
As per google translate: 

"Hello author, the TinyImageNet model you provided here cannot achieve the performance of the paper on both closed and open sets"

Please post further comments in English.

More information is needed before I can help with this issue, specifically:

- How did you train the model? Number of epochs, learning rate, batch size
- What closed-set and open-set performance are you getting?

**Comment by dimitymiller** [comment]
Closing after no further information received.
